ibm/data_source_ibm_is_vpn_gateway_connections.go

```diff
@@ -0,0 +1,234 @@
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	isvpnGatewayConnections       = "connections"
+	isVPNGatewayConnectionsStatus = "status"
+	isVPNGatewayID                = "vpn_gateway_id"
```

Comment:
isVPNGatewayID                = "vpn_gateway_id" change to isVPNGatewayID                = "vpn_gateway" to be inline with reosurces and other vpc reosurces

Reply:
simialry in yesyterday PR for gateway datasource can you please resource_group_id to resource_group because we used same name for all our VPC resources